src/main/java/com/hubspot/jinjava/lib/fn/Functions.java

```diff
@@ -386,15 +388,21 @@ public static int movePointerToJustBeforeLastWord(int pointer, String s) {
     "With two parameters, the range will start at the first value and increment by 1 up to (but not including) the second value. " +
     "The third parameter specifies the step increment. All values can be negative. Impossible ranges will return an empty list. " +
     "Ranges can generate a maximum of " +
-    RANGE_LIMIT +
-    " values.",
+    DEFAULT_RANGE_LIMIT +
+    " values by default, but this integer value is configurable.",
     params = {
       @JinjavaParam(value = "start", type = "number", defaultValue = "0"),
       @JinjavaParam(value = "end", type = "number"),
       @JinjavaParam(value = "step", type = "number", defaultValue = "1")
     }
   )
   public static List<Integer> range(Object arg1, Object... args) {
+    int rangeLimit = requireNonNull(
+        JinjavaInterpreter.getCurrent(),
```

Comment:
I think it's only possible for this to be null in tests if you forget to push an interpreter onto the stack.

Reply:
I remove it, I just don't like when my IDE is screaming at me when something can be null.